Sources/tuist/main.swift

```diff
@@ -8,4 +9,7 @@ if CommandLine.arguments.contains("--verbose") { try? ProcessEnv.setVar(Constant
 LogOutput.bootstrap()
 
 import TuistKit
-TuistCommand.main()
+
+try AsyncQueue.run {
+    TuistCommand.main()
+}
```

Comment:
@pepibumur I have to remove this as it fails some acceptance tests (I'm looking at the scaffolding ones).
Could you explain me what you had in mind with this code here? Thanks!

Reply:
Link to failed AT: https://github.com/tuist/tuist/pull/1930/checks?check_run_id=1336080654